cvat/apps/iam/templates/account/email/email_confirmation_message.html

```diff
@@ -123,11 +133,10 @@
             <td align="center" valign="bottom">
               <a href="https://www.cvat.ai/" target="_blank" style="display: inline-block;">
                 <img
-                  src="https://github.com/cvat-ai/cvat/blob/develop/site/content/en/images/cvat_poster_with_name.png?raw=true"
+                  src="https://app.cvat.ai/static/logo.svg"
```

Comment:
It doesn't seem appropriate to link back to `app.cvat.ai` for emails that could be coming from any CVAT instance... do you think you could change it to link to the domain of the current instance?

(Even better would be to attach the image to the email, but I don't know if that's possible in our implementation.)

Reply:
> Even better would be to attach the image to the email

Convertation of the logo to base64 should work this way

Reply:
It only works for HTML templates, but not for emails. 

In email it would be incorrect URL like this:
<img width="581" alt="image" src="https://github.com/user-attachments/assets/162ae951-3f51-4ffd-aa8c-25a29f11ab24" />

Moreover, context for this template passed not from our code, so
That means that we can't support dynamic logo file, configured in settings (LOGO_FILENAME) without overriding default implementation of `send_confirmation_mail` in `DefaultAccountAdapter`.

Reply:
> That means that we can't support dynamic logo file, configured in settings (LOGO_FILENAME) without overriding default implementation of `send_confirmation_mail` in `DefaultAccountAdapter`.

I think we only need to override [`get_context_data`](https://codeberg.org/allauth/django-allauth/src/commit/4248e0532da93b26e0fcabb9faa5850f4dd7a62b/allauth/account/views.py#L320).

Reply:
Oh, never mind that, I mistook the purpose of that function...

Well I don't know then, maybe we should just remove the logo.